src/views/newCollection/NewCollection.js

```diff
@@ -0,0 +1,27 @@
+import AboutCollection from "components/aboutCollection/AboutCollection";
+import Banner from "components/banner/Banner";
+import FluidGrid from "components/fluidGrid/FluidGrid";
+import importAll from "components/ImportAll";
+import { useEffect } from "react";
+
+const imagesBanner = importAll(
+  require.context("assets/newCollection/Banner", false, /\.(png)$/)
+);
+
+const description = "Soon ...";
+
+const NewCollection = () => {
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
+
+  return (
+    <>
+      <Banner images={imagesBanner} />
+      <AboutCollection description={description} />
```

Comment:
Nie mozesz przeniesc tego bezposrednio do komponentu? Dlaczego to znajduje sie na tym poziomie? 🤔

Reply:
mam na myśli `description`

Reply:
no stwierdziłem, że łatwiej będzie znaleźć to w głownym komponencie jak będzie trzeba zmienić, a po drugie może się przyda taki komponent jeszcze :)